src/libLookup/Lookup.cpp

```diff
@@ -3662,7 +3659,48 @@ void Lookup::SenderTxnBatchThread() {
   DetachedFunction(1, main_func);
 }
 
-void Lookup::SendTxnPacketToNodes(uint32_t numShards) {
+void Lookup::RectifyTxnShardMap(const uint32_t oldShardSize,
+                                const uint32_t newShardSize) {
+  LOG_MARKER();
+
+  auto t_start = std::chrono::high_resolution_clock::now();
```

Comment:
Should we put the performance measuring code inside preprocessor flag?

Reply:
This process should not be invoked very frequently and only on lookup, so it would be nice if we could have real time stats